cmd/root.go

```diff
@@ -16,7 +16,7 @@ import (
 var RootCmd = &cobra.Command{
 	Use:   BinaryName,
 	Short: "A friendly command-line interface to Exercism.",
-	Long: `A command-line interface for the v2 redesign of Exercism.
+	Long: `A command-line interface for the v3 redesign of Exercism.
```

Comment:
I'm wondering if we should even be mentioning the version here, but for now it's as least better than the current text :)